lib/ocpp/v201/charge_point.cpp

```diff
@@ -824,9 +846,10 @@ void ChargePoint::update_aligned_data_interval() {
     const auto next_timestamp = this->get_next_clock_aligned_meter_value_timestamp(
         this->device_model->get_value<int>(ControllerComponentVariables::AlignedDataInterval));
     if (next_timestamp.has_value()) {
+        bool transaction_active = false;
         EVLOG_debug << "Next meter value will be sent at: " << next_timestamp.value().to_rfc3339();
         this->aligned_meter_values_timer.at(
-            [this]() {
+            [this, &transaction_active]() {
```

Comment:
I don't think this is safe to do because at the time the lambda function will be called the boolean will be out of scope.